ocsgw/config/logback.dev.xml

```diff
@@ -1,26 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<configuration>
+<configuration scan="true" scanPeriod="30 seconds" >
 
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
     <encoder>
       <pattern>%d{dd MMM yyyy HH:mm:ss,SSS} %-5p %c{1} - %m%n</pattern>
     </encoder>
   </appender>
 
+  <appender name="CLOUD" class="com.google.cloud.logging.logback.LoggingAppender">
+    <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+      <level>DEBUG</level>
+    </filter>
+    <log>application.log</log>
```

Comment:
`application.log` is the default value.  Better to change it to `ocsgw`.
Then we can use this expression to filter in Stackdriver logging:
`logName="projects/<gcp project>/logs/ocsgw"`